save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/RegistrationView.kt

```diff
@@ -221,13 +221,16 @@ class RegistrationView : AbstractView<RegistrationProps, RegistrationViewState>(
         "PARAMETER_NAME_IN_OUTER_LAMBDA",
     )
     private fun ChildrenBuilder.renderInputForm() {
+        // google does not provide us login, only e-mail, so need to trim everything after "@gmail"
+        val rawInput = state.fieldsMap[InputTypes.USER_NAME] ?: ""
+        val atIndex = rawInput.indexOf('@')
```

Comment:
do we forbid manual creating the name with `@`?

Reply:
yes, moreover - you even cannot input "@" into the input field